README.md

```diff
@@ -135,10 +135,10 @@ All tests should pass.
 To mutate all of the `.cc` files in the library use the following command:
 
 ```
-${DREDD_EXECUTABLE} -p build/compile_commands.json math/src/*.cc --mutation-info-file mutant-info.json
+${DREDD_EXECUTABLE} -p build math/src/*.cc --mutation-info-file mutant-info.json
 ```
 
-The `-p` option allows the compilation database generated by CMake above to be passed to Dredd.
+The `-p` option allows the compilation database generated by CMake above to be passed to Dredd: the `compile_commands.json` file in build will be used as a compilation database.
```

Comment:
I'm being pedantic, but can you put backticks around `build` when you say "... in `build` will be used"?

Reply:
Fixed on [0aa58c8](https://github.com/mc-imperial/dredd/pull/212/commits/0aa58c8db555f37521b51dd6ec3d3a368840a906)